electron/gog/games.ts

```diff
@@ -44,6 +44,14 @@ const installedGamesStore = new Store({
   name: 'installed'
 })
 
+function verifyProgress(stderr: string): boolean {
+  let index = stderr.lastIndexOf('\n')
+  index = stderr.lastIndexOf('\n', index - 1)
+  const status = stderr.substring(index)
+  const match = status.match(/Progress: ([0-9.]+) ([0-9]+)\/([0-9]+)/)
+  return match != null && 100 === Number(match[1]) && match[2] === match[3]
```

Comment:
Small thing here but let's use `!==` just to keep the pattern. 👍🏽

Reply:
Also, can you add a few comments to this function to explain what it is doing? In case we need to do maintenance in the future.

Reply:
actually `!= null` is idiomatic javascript way of checking if value is not null or undefined

Reply:
hmm, `match` will return an array or `null`
So this line could maybe be written like:
`return match && Number(match.join('')) === 100`
If a `match` is `null` it will return `null` which is ok since its a `falsy` value.
What do you think?
I believe its easier to read as well.

Reply:
Changed to `!==`. I still prefer always checking with `!=` because standard library is inconsistent with returning null or undefined. I'm not changing to just `match` though, because falsy is not boolean and I like to be strict about it

Reply:
Ok, that works as well.
Using `Boolean(match)` would convert the null to false in this case.
but it's ok to keep it that way imo.